app/Config/DocTypes.php

```diff
@@ -30,4 +30,19 @@ class DocTypes
         'xhtml-rdfa-1'      => '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">',
         'xhtml-rdfa-2'      => '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.1//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-2.dtd">',
     ];
+
+    /**
+     * Endline HTML Tag
+     *
+     * Generate endline HTML Tag by Document Type
+     *
+     * Currently :
+     * - html5
+     * - Older HTML like `xhtml`
+     *
+     * For :
+     * - `<input>`
+     * - `<img>`
+     */
+    public string $endlineTag = 'html5';
```

Comment:
This setting is for html and form helpers. I hope to add that in the comment.

I would like to change it to `$xhtml = false`. Because it just has two state.

Reply:
Makes sense, thank you for suggest